scripts/README.md

```diff
@@ -0,0 +1,34 @@
+# Helper script documentation
+
+## generate_csv_for_emails.py
+
+This script is used to take input from the following 2 files:
+ - Print file (generated by the actionexporter service and contains enrolment codes)
+ - Respondent/Business excel file (generated by the business and contains contact details and the email 
+ addresses we're sending to)
+
+In addition, the script takes a number of arguments:
+- Survey name.  Currently only 'construction' is valid but more will be in the future.  This name links to a mapping
+with the proper survey name and phone number for the survey.
+- Respondent period (Takes the form of XXX)
+- Return by date (Takes the form of YYY)
+
+
+A csv file is generated from these that will be uploaded to gov.notify(https://www.notifications.service.gov.uk/)
+to send the ras-email-enrolment-code email.  This is currently being done as a special case during the covid-19 situation
+as businesses aren't able to receive their enrolment code for surveys as they're not present at their registered office
+address.
+
+To run the script, do the following:
+```bash
+cd scripts
+pipenv run python generate_csv_for_emails.py <business_filename> <print_filename> construction XXX YYY
```

Comment:
I would be more specific here about what XXX and YYY are rather than expecting people to look a couple of paragraphs up. I'd also consider taking out the "construction" parameter until there's a need to do this for different surveys (only because we're currently on 6 args which is pretty brutal to use :D )

Reply:
Good spot.  That was fully my intention and I got the answer but forgot to update this ^^